src/Config/Postgres/TcpConnectionConfig.php

```diff
@@ -15,22 +15,26 @@
 
 class TcpConnectionConfig extends ConnectionConfig implements ProvidesSourceString
 {
+    public int $port;
```

Comment:
Please add `positive-int` annotation in this cases

Reply:
Added